apps/mobile-wallet/src/screens/LandingScreen.tsx

```diff
@@ -15,73 +15,41 @@ GNU Lesser General Public License for more details.
-import { useSafeAreaInsets } from 'react-native-safe-area-context'
-import styled, { ThemeProvider, useTheme } from 'styled-components/native'
+import { StatusBar } from 'react-native'
+import Animated, { useAnimatedStyle, useSharedValue, withRepeat, withTiming } from 'react-native-reanimated'
+import styled, { useTheme } from 'styled-components/native'
 
 import { sendAnalytics } from '~/analytics'
 import AppText from '~/components/AppText'
 import Button from '~/components/buttons/Button'
-import Screen, { ScreenProps } from '~/components/layout/Screen'
 import { useAppDispatch, useAppSelector } from '~/hooks/redux'
-import altLogoSrc from '~/images/logos/alephiumHackLogo.png'
-import AlephiumLogo from '~/images/logos/AlephiumLogo'
 import RootStackParamList from '~/navigation/rootStackRoutes'
 import { storedWalletExists } from '~/persistent-storage/wallet'
 import { methodSelected, WalletGenerationMethod } from '~/store/walletGenerationSlice'
-import { BORDER_RADIUS_BIG, BORDER_RADIUS_HUGE } from '~/style/globalStyle'
-import { themes } from '~/style/themes'
 import { resetNavigation } from '~/utils/navigation'
 
-interface LandingScreenProps extends StackScreenProps<RootStackParamList, 'LandingScreen'>, ScreenProps {}
-
-const gradientColors = ['#eb3d55', '#ff8772', '#ffcd61', '#3a99e2', '#012a6c', '#000044']
-const gradientAltColors = ['#1effd6', '#fbe201', '#fb01e6', '#3900aa', '#05064f', '#05064f']
+interface LandingScreenProps extends StackScreenProps<RootStackParamList, 'LandingScreen'> {}
 
-const LandingScreen = ({ navigation, ...props }: LandingScreenProps) => {
+const LandingScreen = ({ navigation }: LandingScreenProps) => {
   const dispatch = useAppDispatch()
-  const insets = useSafeAreaInsets()
-  const theme = useTheme()
   const { t } = useTranslation()
-  const isWalletUnlocked = useAppSelector((s) => s.wallet.isUnlocked)
+  const theme = useTheme()
+  const isWalletUnlocked = useAppSelector((state) => state.wallet.isUnlocked)
 
-  const { width, height } = Dimensions.get('window')
-  const [dimensions, setDimensions] = useState({ width, height })
   const [isScreenContentVisible, setIsScreenContentVisible] = useState(false)
+  const rotation = useSharedValue(0)
 
   useFocusEffect(
     useCallback(() => {
       storedWalletExists()
         .then((walletExists) => {
           if (walletExists) {
-            // Normally, when the app is unlocked, this screen is not in focus. However, under certain conditions we end
-            // up with an unlocked wallet and no screen in focus at all. This happens when:
-            // 1. the auto-lock is set to anything but "Fast"
-            // 2. the user manually kills the app before the auto-lock timer completes
-            // 3. the WalletConnect feature is activated
-            // Since there is no screen in focus and since the default screen set in the RootStackNavigation is this
-            // screen, we need to navigate back to the dashboard.
```

Comment:
why removing all these comments? I think it's important to leave them there as it is not self-evident what is happening here by just reading the code.